From 8f260e45bd6d9056914a3611cc7c1d934c43f9fb Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@freescale.com>
Date: Fri, 9 Nov 2012 19:07:36 +0000
Subject: [PATCH 387/518] dpa_classifier: Use "bool" for boolean attributes
 instead of "int"

"bool" is available both in Linux user space and in Linux kernel space.

Signed-off-by: Marian Chereji <marian.chereji@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 .../staging/fsl_dpa_offload/dpa_classifier_ioctl.h |    2 +-
 include/linux/fsl_dpa_classifier.h                 |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h b/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h
index 9dd3848..adb92be 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h
+++ b/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h
@@ -203,7 +203,7 @@ struct dpa_cls_compat_tbl_enq_action_desc {
 
 struct dpa_cls_compat_tbl_action {
 	enum dpa_cls_tbl_action_type	type;
-	int				enable_statistics;
+	bool				enable_statistics;
 	union {
 		struct dpa_cls_compat_tbl_enq_action_desc enq_params;
 		struct dpa_cls_tbl_next_table_desc	  next_table_params;
diff --git a/include/linux/fsl_dpa_classifier.h b/include/linux/fsl_dpa_classifier.h
index d466bee..6f58903 100644
--- a/include/linux/fsl_dpa_classifier.h
+++ b/include/linux/fsl_dpa_classifier.h
@@ -150,7 +150,7 @@ struct dpa_cls_tbl_exact_match_params {
 	uint8_t		key_size;
 
 	/* Use priorities for each entry in table if nonzero */
-	int		use_priorities;
+	bool		use_priorities;
 };
 
 /* DPA Classifier table parameters */
@@ -268,7 +268,7 @@ struct dpa_cls_tbl_action {
 	enum dpa_cls_tbl_action_type	type;
 
 	/* Enable statistics for this entry if nonzero */
-	int				enable_statistics;
+	bool				enable_statistics;
 
 	union {
 
-- 
1.7.5.4

